macos/FluentUI/Link.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+
+/// A lightweight hyperlink control
+@objc(MSFLink)
+open class Link : NSButton {
+	
+	/// Initialies a link with a content string that is displayed, and an underlying
+	/// - Parameters:
+	///   - content: The visible text of the link that the user sees.
+	///   - url: The URL that is opened when the link is clicked
+	@objc public init(content: String, url: NSURL) {
```

Comment:
Not sure how I feel about the "content" name for the string. Buttons, which this is, have titles, not "content". https://developer.apple.com/documentation/appkit/nsbutton/1524430-title

Reply:
I can look into what it would take to keep it as the property "title", and then have React Native map the "content" prop to the native "title" property

Reply:
I removed the "content" property altogether, and just called the function parameter "title"